src/css/index.scss

```diff
@@ -0,0 +1,156 @@
+body {
+  margin: 0;
+  border: 0;
+  padding: 0;
+}
+
+.bui-icon,
+address,
+cite,
+dfn,
+em,
+i,
+var {
+  font-style: normal;
+}
+
+/* .table {
+  border: 0;
+  table-layout: fixed;
+  width: 300px;
+  margin-top: 36px;
+} */
+
+.fieldset {
+  border: 0px;
+}
+
+.data-border {
+  width: 52px;
+  height: 7px;
+  background-color: rgb(223, 49, 67);
+  margin-bottom: 8px;
+  margin-left: 5px;
+}
+
+.info-section {
+  flex: 1;
+  width: 50%;
+  height: 521px;
+  min-width: 490px;
+  font-size: 16px;
+  box-sizing: border-box;
+}
+
+.info-container {
+  overflow: hidden;
+  width: 100%;
+  min-width: 980px;
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+}
+
+#quick-id-td td {
+  height: 40px;
+  font-size: 21px;
+}
+
+.td-lable {
+  width: 150px;
+}
+
+.td-content {
+  width: 250px;
+}
+
+.info-section a {
+  color: white;
+  text-decoration: none;
+}
+
+.info-title {
+  letter-spacing: 3.8px;
+  font-size: 36px;
+  font-weight: 600;
+}
+
+#info-quick-id {
+  background-color: rgb(46, 73, 117);
+  color: white;
+  padding-top: 80px;
+  padding-left: 25%;
+  margin-right: -20%;
```

Comment:
This layout method with padding and margin is not working on laptop solutions.

This layout part will be in the assignment 3 after we learn the layout in lesson 3.